src/config.c

```diff
@@ -2191,6 +2207,20 @@ static int updateHZ(long long val, long long prev, const char **err) {
     return 1;
 }
 
+static int updatePort(long long val, long long prev, const char **err) {
+    /* Do nothing if port is unchanged */
+    if (val == prev) {
+        return 1;
+    }
+
+    if (changeListenPort(val, &server.ipfd, acceptTcpHandler) == C_ERR) {
+        *err = "Unable to listen on this port. Check server logs.";
```

Comment:
I don't see new information added to the logs in `changeListenPort`, so perhaps change the message to something like:

"Unable to bind to port %d."

Reply:
i suppose it refers to the log print in `listenToPort`